textrepo-app/src/main/java/nl/knaw/huc/service/task/DeleteContents.java

```diff
@@ -21,11 +21,14 @@ public DeleteContents(String contentsSha) {
 
   @Override
   public void executeIn(Handle transaction) {
+    var savepoint = "delete-" + contentsSha;
```

Comment:
Gegeven hoe belangrijk dit savepoint is, voor de zekerheid 'final' maken?

Reply:
Gefixt in 28a47de